src/parser/semantic_check.c

```diff
@@ -11429,6 +11430,17 @@ pt_check_with_info (PARSER_CONTEXT * parser, PT_NODE * node, SEMANTIC_CHK_INFO *
       sc_info_ptr->system_class = false;
       node = pt_resolve_names (parser, node, sc_info_ptr);
 
+      if (!pt_has_error (parser))
+	{
+	  if ((node->node_type == PT_INSERT && node->info.insert.spec->info.spec.remote_server_name)
```

Comment:
How about adding PT_INSERT_DBLINK, PT_DELETE_DBLINK, PT_UPDATE_DBLINK, PT_MERGE_DBLINK?

Reply:
Good point, but I will keep the logic to prevent from accidental side effect.